spec/models/program_spec.rb

```diff
@@ -253,10 +253,28 @@
   end
 
   describe '#remaining_cfp_types' do
-    it 'returns an array with the types for which a cfp doesn\'t exist' do
+    it 'returns an array with the types for which a cfp doesn\'t exist, when only the Event type does' do
       expect(program.remaining_cfp_types).to eq(Cfp::TYPES)
       create(:cfp, cfp_type: 'events', program: program, end_date: Date.current + 1)
+      expect(program.remaining_cfp_types).to eq(['booths'])
+    end
+
+    it 'returns an array with the types for which a cfp doesn\'t exist, when only the Booth type does' do
+      expect(program.remaining_cfp_types).to eq(Cfp::TYPES)
+      create(:cfp, cfp_type: 'booths', program: program, end_date: Date.current + 1)
+      expect(program.remaining_cfp_types).to eq(['events'])
+    end
+
+    it 'returns an empty array when all the cfp types exist' do
+      expect(program.remaining_cfp_types).to eq(Cfp::TYPES)
+      create(:cfp, cfp_type: 'events', program: program, end_date: Date.current + 1)
```

Comment:
Why do we need to set a specific end_date? Does the date matter? Isn't the default end_date from the factory enough?